frontend/src/components/dashboard/aliases/AliasGenerationButton.tsx

```diff
@@ -372,7 +372,6 @@ const AliasTypeMenuItem = (props: AliasTypeMenuItemProps) => {
     {
       key: props.item.key,
       isDisabled: props.item.isDisabled,
-      onAction: props.onAction,
```

Comment:
I think you should also be able to remove line 365 now.

Reply:
nice catch. Removed this (and [the corresponding reference to this prop on line 346 too](https://github.com/mozilla/fx-private-relay/pull/5038/files#diff-621cb0db2d20dbbe0fbcd052dd4a0d05a2b62291aceccf9a1f8a0f96c6fdc734L346)!)